### PR TITLE
Convert date to number in comparizon operators

### PR DESCRIPTION
**Pull request by DeLaGuardo**

### CODE REVIEW

File: src/axel_f/excel/date.cljc [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
+                   java.time.ZoneId)))
+
+#?(:clj (defmethod coerce/excel-number java.time.LocalDate [ld]
+          (.toEpochSecond (.atStartOfDay ld (ZoneId/ofOffset "UTC" (ZoneOffset/ofHours 0))))))
+
+#?(:clj (defmethod coerce/excel-number java.time.LocalDateTime [ldt]
+          (.toEpochSecond ldt ZoneOffset/UTC)))
+
+#?(:cljs (defmethod coerce/excel-number js/Date [jsd]
+           (Math/round (/ (.getTime jsd) 1000))))
```
Comment (mreinsch) [inline src/axel_f/excel/date.cljc]: why divided by 1000?

File: test/axel_f/date_test.cljc [hunk 2]
```diff
@@ -55,3 +55,15 @@
...
              (sut/YEAR (sut/DATE 1988 8 21))
              ((af/compile "YEAR(DATE(1988, 8, 21))"))))))
+
+(t/deftest comparable
+  (t/testing "local date"
+    (t/is ((af/compile "DATE(1988, 8, 21) > DATE(1988, 7, 21)")))
+    (t/is ((af/compile "DATE(1988, 8, 21) = DATE(1988, 8, 21)")))
+    (t/is ((af/compile "DATE(1988, 8, 21) < DATE(1988, 9, 21)")))
+    (t/is ((af/compile "TODAY() > DATE(1988, 8, 21)")))
+    (t/is ((af/compile "TODAY() > 0"))))
```
Comment (mreinsch) [inline test/axel_f/date_test.cljc]: and let's add some more tests, like 
```
DATE(2020, 02, 03, 11, 03, 29) < 1580727810
DATE(2020, 02, 03, 11, 03, 30) = 1580727810
DATE(2020, 02, 03, 11, 03, 31) = 1580727810
```

File: src/axel_f/excel/date.cljc (hunk 1 above)
Comment (DeLaGuardo) [reply src/axel_f/excel/date.cljc]: because javascript gives you millis

File: test/axel_f/date_test.cljc (hunk 2 above)
Comment (DeLaGuardo) [reply test/axel_f/date_test.cljc]: this isn't a valid examples. atm we don't have function to instantiate datetime, only `NOW()` can give you datetime

**Pull request merged**